File: Go/String/1152.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	var input string
	reader := bufio.NewReader(os.Stdin)
	//bufio의 NewReader로 standard input을 받는 입력 인스턴스 kbReader를 생성합니다.
	input, _ = reader.ReadString('\n') //ReadString('\n')에서는 엔터키를 받을 때까지 키보드의 입력을 받습니다.
	//한줄 입력 받기 , 뜨어쓰기 포함

	words := strings.Split(input, " ") //해당 문자열을 " "을 기준으로 잘라 배열 형태로 저장
	var count int
	for i := range words {
		if words[i] != "\n" && words[i] != "" { //비어있거나 엔터 입력아니면 문자로 취급하여 단어 저장
			count++
		}
	}
	fmt.Println(count)
}

File: python/String/1152.py
if __name__=="__main__":
    str_in=input() #입력 받고 
    list_str=str_in.split(" ") # " "띄어쓰기로 구분하여 리스트를만듬
    result=0 
    for val in list_str:
        if(val!=''): # 만약 list 해당 값임 빈칸이 아니면 단어이기 때문에 +1 
            result+=1
    print(result)
